src/main/java/com/osiris/dyml/YamlSection.java

```diff
@@ -166,41 +166,52 @@ public YamlSection addKeys(String... keys) {
         return this;
     }
 
+    /**
+     * See {@link #addValues(List)} for details.
+     */
+    public <T> YamlSection addValues(T... v) {
+        if (v != null)
+            return addValues(utils.arrayToValuesList(v));
+        else
+            return addValues((List<SmartString>) null);
+    }
```

Comment:
you can keep "return this",
also keep it in all the other functions
its just to make it more obvious whats returned